scripts/variantstore/tieout/GIAB_TIEOUT.md

```diff
@@ -0,0 +1,175 @@
+# NOTES
+ - Previously there was an attempt to tie out the feature extract input to VQSR between WARP and GVS.  However, that was abandoned in favor of the truth-sample based comparison here due to non-determinism and bugs in GenomicsDB that were hard to compensate for.  However, the docs and scripts for doing this are in the branch `gvs_vqsr_input_tieout` under `scripts/variantstore/tieout/feature_extract`
+ 
+## Prerequisites
+
+ - rtg
+ - bcftools
+ - tabix
+ - python 3.7+
+
+## One time tasks
+```
+
+# create SDF for reference (used by rtg eval)
+# use local path to GRC38 reference fasta
+REFERENCE="/Users/kcibul/projects/references/hg38/v0/Homo_sapiens_assembly38.fasta"
+rtg format --output human_REF_SDF $REFERENCE
+
+# copy down truth data locally
+mkdir -p truth && gsutil -m cp gs://broad-dsp-spec-ops/gvs/truth/* truth/
+```
+
+## Obtain Truth sample VCFs 
+
+First, create a full cohort extract (as described in README.md) using the desired filter_set_name.  Assuming this is in a single gathered VCF of `gvs.vcf.gz`
```

Comment:
I would appreciate being hit over the head about where in the Readme to look....I am assuming I want to run: Running ExtractCohort against BQ ?!?! but that's in the debugging section and that has no `filter_set_name`